examples/prometheus-alerting-rules/alerts.yaml

```diff
@@ -3,9 +3,7 @@ groups:
   rules:
   - alert: KubeStateMetricsListErrors
     annotations:
-      description: kube-state-metrics is experiencing errors at an elevated rate in
-        list operations. This is likely causing it to not be able to expose metrics
-        about Kubernetes objects correctly or at all.
+      description: kube-state-metrics is experiencing errors at an elevated rate in list operations. This is likely causing it to not be able to expose metrics about Kubernetes objects correctly or at all.
```

Comment:
@KlavsKlavsen  can you remove the changes to examples/ as well? CI is failing due to this I think.

Reply:
My bad - sorry. Fixed :)